charts/canvas-oda/values.yaml

```diff
@@ -131,7 +131,7 @@ controller:
     
 dependentapi-simple-operator:
   enabled: true
-  image: mtr.devops.telekom.de/magenta_canvas/public:dependentapi-simple-operator-0.1.0
+  image: ankursarkar/dependentapi-simple-operator:0.2.0-rc
```

Comment:
For the pull request to master branch, we should use a fixed version, not a release-candidate (*-rc), which might be overwritten by some pipeline.

Reply:
-rc has been remove from the build.sh & build.cmd